packages/babel-plugin-transform-es2015-template-literals/src/index.js

```diff
@@ -18,7 +18,10 @@ export default function ({ types: t }) {
         let raw = [];
 
         for (const elem of (quasi.quasis: Array)) {
-          strings.push(t.stringLiteral(elem.value.cooked));
+          const value = elem.value.cooked == null
+            ? t.identifier("undefined")
```

Comment:
There could be a `var undefined = 4` in the top scope of the module so it would break. You could also use `path.scope.buildUndefinedNode()`